### ISSUE TITLE
[BUG]  _PyEval_EvalFrameDefault returned NULL

### ISSUE THREAD

**Issue by frymex** [dashingsoft/pyarmor#898]
when running my program (obfuscation) i get an error - SystemError: _PyEval_EvalFrameDefault returned NULL without setting an error

but at the same time, I don't have any errors on my machine and the software runs normally.

Virtual machine characteristics:
Ubuntu 22.04

My pc specs:
Windows 11

**Comment by alireza-heidari** [comment]
i had the same issue when i run program with docker

**Comment by ikaranmali** [comment]
Yes I am getting same error in Debian 11, the obfuscation is done in ubuntu image.

**Comment by jondy** [comment]
I close it because I have no way to reproduct it.
If anyone occurs this issue, please report with necessary information
